example_project_bzl_4/third_party/readline/readline.nix

```diff
@@ -0,0 +1,57 @@
+let pkgs = import <nixpkgs> {};
```

Comment:
I would suggest using [overrideAttrs](https://nixos.org/manual/nixpkgs/stable/#sec-pkg-overrideAttrs) pattern, to remove code duplication between nixpkgs channel and this repository - it helps in codifying only the difference you are interested in:
```
# oldAttrs is the definition from before 
pkgs.readline.overrideAttrs (oldAttrs: rec {
  name = "readline-8.1p0"
  # Override source
  src = pkgs.fetchurl {
     ...
  };
  # Get rid of multiple outpus
  outputs = [ "out"];
  # Extend configuration flags
  configureFlags = oldAttrs.configureFlags + [
    "--enable-static"
    "--disable-shared"
  ];
  # remove unwanted patch
  patches =  pkgs.lib.lists.drop 1 oldAttrs.patches;
  # update meta branch
  meta.branch = "8.1";
});
```

Reply:
Hm... awesome thanks!

Reply:
Done :)